client/src/App.js

```diff
@@ -18,24 +24,46 @@ const theme = createMuiTheme({
     },
 });
 
+function PostsApp() {
```

Comment:
This was created to separate views, by one side you have sign in and sign up, in the other side the rest of the app with the navbar.